test/run/ch-run_join.bats

```diff
@@ -187,9 +203,8 @@ unset_vars () {
     ch-run -v --join-ct=3 --join-tag=foo "${ch_timg}" -- \
            /test/printns 0 "${BATS_TMPDIR}/join.2.ns" \
            >& "${BATS_TMPDIR}/join.2.err" &
-    sleep 1
+    sleepcat 6 "${BATS_TMPDIR}/join.2.ns"
```

Comment:
why are some `sleepcat 6` while others `sleepcat 3` ?

Reply:
The 6 is to account for up to five retries on the namespace join. 3 is just “wait a little while” — two iterations of the `sleepcat` wait loop IIRC?